compute/cloud-client/src/main/java/compute/ipaddress/AssignStaticExistingVm.java

```diff
@@ -52,24 +52,33 @@ public static void main(String[] args)
   public static Instance assignStaticExistingVmAddress(String projectId, String instanceId,
                                                        String zone, String netInterfaceName)
           throws IOException, ExecutionException, InterruptedException, TimeoutException {
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests. After completing all of your requests, call
+    // the `instancesClient.close()` method on the client to safely
+    // clean up any remaining background resources.
```

Comment:
Remove the second line in the comment. Kindly follow this practice across samples in all PRs.